tests/test_rotate_visualisations.py

```diff
@@ -0,0 +1,128 @@
+from tests.helper import requires
+from swiftsimio.subset_writer import find_links, write_metadata
+from swiftsimio import mask, cosmo_array, load
+from swiftsimio.visualisation.projection import project_gas
+from swiftsimio.visualisation.slice import slice_gas
+from swiftsimio.visualisation.volume_render import render_gas
+from swiftsimio.visualisation.rotation import rotation_matrix_from_vector
+import h5py
+from numpy import array_equal, mean
+
+@requires("cosmological_volume.hdf5")
+def test_rotation(filename):
+    output = "single_particle.hdf5"
+    
+    data_mask = mask(filename)
+    boxsize = data_mask.metadata.boxsize
+    region = [[0, b] for b in boxsize]
+    data_mask.constrain_spatial(region)
+    
+    # Write the metadata
+    infile = h5py.File(filename, "r")
+    outfile = h5py.File(output, "w")
```

Comment:
Can we use an in-memory HDF5 file here, like we do for the other tests? There are helper functions for doing that in the test helpers.